gwtp-core/gwtp-mvp-client/src/main/java/com/gwtplatform/mvp/client/HasSlots.java

```diff
@@ -38,6 +48,94 @@
  */
 public interface HasSlots {
     /**
+    * This method adds some content in a specific slot of the {@link Presenter}.
+    * The attached {@link View} should manage this slot when its
+    * {@link View#addToSlot(Object, com.google.gwt.user.client.ui.Widget)} is called.
+    * <p/>
+    * Contrary to the {@link #setInSlot} method, no
+    * {@link com.gwtplatform.mvp.client.proxy.ResetPresentersEvent} is fired,
+    * so {@link PresenterWidget#onReset()} is not invoked.
+    * <p/>
+    * For more details on slots, see {@link HasSlots}.
+    *
+    * @param slot An opaque object identifying which slot this content is being
+    * added into.
+    * @param content The content, a {@link PresenterWidget}. Passing {@code null}
+    * will not add anything.
+    */
+    @Deprecated
```

Comment:
Can you add the `@deprecated` tag to the javadoc too and just write up something on how to migrate? Could also point to a page on the wiki.